.github/workflows/test.yaml

```diff
@@ -70,24 +70,26 @@ jobs:
             fail-fast: false
             matrix:
                 php-version: ['8.1', '8.3', '8.4']
+                symfony-version: ['6.4', '7.2']
```

Comment:
this should not have a `7.2` version IMO. The default should be to run with the unmodified dependencies.

And I suggest running tests against 6.4 a single PHP version (and only against highest dependencies, as it probably also make sense to run `lowest` only once with the unmodified lowest dependencies)

Reply:
This is something we can do, I will rewrite the matrix entries for something like https://github.com/Kocal/BiomeJsBundle/blob/main/.github/workflows/ci.yaml#L41-L101, in order to have a 100% explicit test suites